Examples/PythonToInflux/client.py

```diff
@@ -0,0 +1,70 @@
+import argparse
+from influxdb import InfluxDBClient
+import datetime
+
+def main(host='localhost', port=8086):
+    """Instantiate a connection to the InfluxDB."""
+    user = 'root'
+    password = 'root'
+    dbname = 'example'
+    dbuser = 'user'
+    dbuser_password = 'my_secret_password'
+    query = 'select value from cpu_load_short;'
+    json_body = [
+        {
+            "measurement": "cpu_load_short",
+            "tags": {
+                "host": "server01",
+                "region": "us-west"
+            },
+            "time": datetime.datetime.now(),
+            "fields": {
+                "Float_value": 0.64,
+                "Int_value": 3,
+                "String_value": "Text",
+                "Bool_value": True
+            }
+        }
+    ]
+
+    client = InfluxDBClient(host, port, user, password, dbname)
+
+    print("Create database: " + dbname)
+    client.create_database(dbname)
+
+    print("Create a retention policy")
+    client.create_retention_policy('awesome_policy', '3d', 3, default=True)
+
+    print("Switch user: " + dbuser)
+    client.switch_user(dbuser, dbuser_password)
+
+    print("Write points: {0}".format(json_body))
+    client.write_points(json_body)
+
+    print("Querying data: " + query)
+    result = client.query(query)
```

Comment:
Might be useful to add a couple sample queries, even if they are just comments i.e.
`query = "SELECT MEAN(value) FROM {} WHERE \
            time > now() - 10d GROUP BY time(500m)".format(metric)`